linux/shim.go

```diff
@@ -33,6 +36,11 @@ func newShim(shimName string, path, namespace string, remote bool) (shim.ShimCli
 	}
 	cmd := exec.Command(shimName, "--namespace", namespace)
 	cmd.Dir = path
+	if debug {
+		cmd.Args = append(cmd.Args, "--debug")
+		cmd.Stdout = os.Stdout
```

Comment:
The problem with doing this is reattach when containerd dies and shims keep running.  Part of my plugins work was moving bundles out of /run and into /var/lib/containerd where we could setup shim logs inside the bundles.